docs/content/users/quickstart.md

```diff
@@ -463,17 +463,18 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
     # Change the base-url below to your project's URL
```

Comment:
Sorry to request other things here, but let's separate the domain name into an environment variable so they can mostly copy/paste. If we say `export MAGENTO_HOSTNAME= my-magento2-site` and then use `mkdir ${MAGENTO_HOSTNAME} && cd ${MAGENTO_HOSTNAME}` then `ddev magento setup:install with the ${MAGENTO_HOSTNAME}.ddev.site` will work without them having to stop and fiddle with it.